docs/api-docs/channels/channels-extending-existing.md

```diff
@@ -103,85 +122,75 @@ Accept: application/json
+| &nbsp;&nbsp; ↳ `title`      | str        | Label displayed to merchants in navigation bar                                                                                                              |
+| &nbsp;&nbsp; ↳ `query_path` | str        | Passed to app's iframe. Ex: `https://<store_url>/manage/channel/2/app?id=5&section=overview`                                                                |
+
+<!-- title: Accepted platforms and types -->
+
+| Platform          | Accepted Type                                   |
+| ----------------- | ----------------------------------------------- |
+| `square`          | `pos`                                           |
+| `vend`            | `pos`                                           |
+| `clover`          | `pos`                                           |
+| `facebook`        | `marketplace`,`marketing`                       |
+| `amazon`          | `marketplace`                                   |
+| `ebay`            | `marketplace`                                   |
+| `wordpress`       | `storefront`                                    |
+| `drupal`          | `storefont`                                     |
+| `acquia`          | `storefront`                                    |
+| `bloomreach`      | `storefront`                                    |
+| `deity`           | `storefront`                                    |
+| `google_shopping` | `marketing`                                     |
+| `custom`          | `storefront`, `pos`, `marketing`, `marketplace` |
 
 For a complete Channel API reference (including request schemas and property descriptions), see: [API Reference > Channels and Listings](https://developer.bigcommerce.com/api-reference/cart-checkout/channels-listings-api).
 
 ## Terminology
 
-|Term|Definition|
-|-|-|
-|**Channels Toolkit**|Channels Toolkit is a  set of tools, UI patterns, guidelines, and APIs provided by BigCommerce to enable partners and developers to extend the BigCommerce ecosystem by building sales channel integrations (whether they are point of sale, marketing, marketplace, social, or headless storefronts) that are more deeply embedded within the BigCommerce control panel.|
-|**Select Partners**|Partners approved by BigCommerce to be marketed and discoverable as sales channel providers within the new Channel Manager. These are partners who offer what we know to be best in breed integrations to key sales channels that help merchants grow their business.|
-
-<div class="HubBlock--callout">
-<div class="CalloutBlock--info">
-<div class="HubBlock-content">
+| Term                 | Definition                                                                                                                                                                                                                                                                                                                                                               |

```

Comment:
Please keep the original version. I think additional dashes may cause issues with how the table is rendered on the site.